packages/core/src/operation.ts

```diff
@@ -40,6 +41,10 @@ export interface OperationFunction<TOut> extends Labelled {
   (task: Task<TOut>): Operation<TOut>;
 }
 
+export interface OperationObject<TOut> extends Record<string | symbol | number, unknown> {
```

Comment:
@wKich I was more raising the question than saying what the answer was. Both `Record<string, unknown>` and `Record<string|symbol|number,unknown>` seem to work. Question is which is more forward compatible? Do you have an opinion?

Reply:
I think `Record<string|symbol|number,unknown>` is better, but also you will not have numbers as a keys, using symbols is pretty rare case. But at least `string|symbol|number` is the most wider type that supported by `Record`

![image](https://user-images.githubusercontent.com/6397708/150472476-b744ef47-3f31-495f-a911-f703514bed49.png)

Reply:
👍🏻